src/api/chrome.js

```diff
@@ -237,14 +237,13 @@ const getBadgeCounter = async (initialValue = 0) => {
 }
 
 export const getCurrentTabUrlBackground = async () => {
-  const currWindow = await chrome.windows.getCurrent()
   // `tab` will either be a `tabs.Tab` instance or `undefined`.
   const [tab] = await chrome.tabs.query({
     active: true,
-    windowId: currWindow.id
+    lastFocusedWindow: true
```

Comment:
as I expected, this is one of the obvious use cases also shown in the docs https://developer.chrome.com/docs/extensions/reference/tabs/#get-the-current-tab